components/service/glean/src/test/java/mozilla/components/service/glean/EventMetricTypeTest.kt

```diff
@@ -0,0 +1,227 @@
+package mozilla.components.service.glean
```

Comment:
@pocmo I landed the test in #1087 with the "public domain" license, as per our [Mozilla Policy](https://www.mozilla.org/en-US/MPL/headers/). I thought that also applied on this repo, since I saw at least [another usage](https://github.com/mozilla-mobile/android-components/blob/210751ad2b02639cbfc729b0e96a0c2dc204e846/components/support/utils/src/test/java/mozilla/components/support/utils/WebURLFinderTest.java). But looking at other files, I found that some tests are licensed as MPL as well. Should I use the public domain or MPL?